packages/headless/docs/actions-parser.ts

```diff
@@ -1,3 +1,39 @@
-export function parseActions() {
-  return {};
+import {Config, DocComment, DocGen, Entity, Module} from './doc-json-types';
+import {getDesc, getModule} from './utils';
+
+export function parseActions(docgen: DocGen, config: Config) {
+  return config.actions.sections.map((section) => {
+    const sectionModules: Module[] = section.sources.map((source) =>
```

Comment:
Is the `Module[]` type necessary to explicitly write-out here? Typescript is able to imply types based on what is to the right of the `=` sign if what is there is already typed.